package.json

```diff
@@ -20,22 +20,32 @@
     "react-dom": "^16.12.0",
     "react-helmet": "^5.2.1",
     "react-i18next": "^11.2.7",
+    "react-load-script": "0.0.6",
     "styled-components": "^4.4.1"
   },
   "devDependencies": {
+    "gatsby-plugin-eslint": "^2.0.8",
     "prettier": "^1.19.1"
   },
   "keywords": [
-    "gatsby", "ReactJS"
+    "gatsby",
+    "ReactJS"
   ],
   "license": "MIT",
   "scripts": {
     "build": "gatsby build",
-    "develop": "gatsby develop",
+    "develop": "gatsby develop --host=0.0.0.0 --https --open -p 8443 --key-file ./src/cert/key.pem --cert-file ./src/cert/cert.pem",
```

Comment:
I use the port 3000 and then I can serve js-payments on 8443 to use it in iframes

Reply:
ok, done